code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2239,3 +2239,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/pruno/on_mob_life(mob/living/carbon/M)
 	M.adjust_disgust(5)
 	..()
+
+/datum/reagent/consumable/ethanol/brass_monkey
+	name = "Brass Monkey"
+	color = "#e9e357"
+	description = "An ancient beverage supposedly invented by a bunch of monkeys trying to join a failing steampunk cult. Traditionally served in a clockwork mug."
+	boozepwr = 85
+	taste_description = "steam-powered bananas"
+	glass_icon_state = "brass_monkey"
+	glass_name = "Brass Monkey"
+	glass_desc = "Coolin' by the lockers, getting kind of funky, me and the crew, we're drinking Brass Monkey."
+
+/datum/reagent/consumable/ethanol/brass_monkey/on_mob_add(mob/living/M)
+	M.faction |= "jungle"
+
+/datum/reagent/consumable/ethanol/brass_monkey/on_mob_delete(mob/living/M)
+	M.faction -= "jungle"
```

Comment:
this will create issues where someone with the jungle faction NOT from the drink loses said faction.

Reply:
Good catch! I'll have to think of a way to check if the jungle faction did or didn't already exist on the mob.